index.html

```diff
@@ -0,0 +1,1209 @@
+            </div>
+          </section>
+
+          <section id="acl-resources" inlist="" rel="schema:hasPart" resource="#acl-resources">
+            <h2 property="schema:name">ACL Resources</h2>
+            <div datatype="rdf:HTML" property="schema:description">
+              <p>This section describes ACL resource <a href="#acl-resource-discovery" rel="cito:discusses">discovery</a> and <a href="#acl-resource-representation" rel="cito:discusses">representation</a>.</p>
+
+              <section id="acl-resource-discovery" inlist="" rel="schema:hasPart" resource="#acl-resource-discovery">
+                <h3 property="schema:name">ACL Resource Discovery</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>When a server wants to enable applications to discover <a href="#authorization">Authorizations</a> associated with a given <a href="#resource">resource</a>, the server MUST advertise the <a href="#acl-resource">ACL resource</a> that is associated with a resource by responding to an HTTP request including a <code>Link</code> header with the <code>rel</code> value of <code>acl</code> (<cite><a href="#acl-link-relation" rel="rdfs:seeAlso">acl Link Relation</a></cite>) and the ACL resource as link target.</p>
+
+                  <p>ACL Resource Discovery is used towards determining the <cite><a href="#effective-acl-resource" rel="rdfs:seeAlso">Effective ACL Resource</a></cite> of a resource.</p>
+
+                  <div class="issue" id="link-relation-type" rel="schema:hasPart" resource="#link-relation-type">
+                    <h4 property="schema:name"><span>Issue</span>: Link Relation Type</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>The possibility of using URIs as relation types interchangeably or as alternate to the <cite><a href="#acl-link-relation" rel="rdfs:seeAlso">acl Link Relation</a></cite> type is under consideration, e.g., <code>https://www.w3.org/ns/iana/link-relations/relation#acl</code> or <code>http://www.w3.org/ns/auth/acl#accessControl</code>: <a href="https://github.com/solid/web-access-control-spec/issues/21#issuecomment-769137284" rel="cito:citesAsRelated">issues/21</a>.</p>
+                    </div>
+                  </div>
+
+                  <p>Servers MUST NOT directly associate more than one ACL resource to a resource.</p>
```

Comment:
What about directly associating one ACL resource to more than one resource?
I think TrinPod does that and I don't think the spec currently forbids it,
but it does makes the application of `acl:Control` quite a bit more complex, so it may be worth discouraging?
cc @gibsonf1

Reply:
Indeed not forbidden.

I held off on elaborating on that for the time being. Would love more feedback based on implementation experience before writing (probably) non-normative text for it. Closely related is the behaviour around deleting resources and whether the protocol (like Solid) requires deletion of the associated ACL resource. That also complicates things. We can mention both and other scenarios but was hoping looking for substantial information. In any case, unless there are WAC-centric security considerations on this, we may not need to mention it either.